.docker/ci/Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY . src/moveit2
 RUN \
     # Update apt package list as previous containers clear the cache
     apt-get -q update && \
-    apt-get -q -y upgrade && \
+    apt-get -q -y upgrade --with-new-pkgs && \
```

Comment:
Do you want to add a TODO or create an issue to remove later? Seems fine either way, to me

Reply:
We might even want to leave it this way. I don't think there is anything wrong with always allowing to install new dependencies if they show up.